distreqx/bijectors/triangular_linear.py

```diff
@@ -77,8 +77,8 @@ def forward(self, x: Array) -> Array:
 
     def forward_log_det_jacobian(self, x: Array) -> Array:
         """Computes log|det J(f)(x)|."""
-        triangular_logdet = jnp.vectorize(_triangular_logdet, signature="(m,m)->()")
-        return triangular_logdet(self._matrix)
+        triangular_logdet = jax.vmap(_triangular_logdet)
+        return triangular_logdet(jnp.expand_dims(self._matrix, axis=0))[0]
```

Comment:
Actually maybe the above is true for all of them? I mean if we only return the first element do we even need to vmap? We can just do the thing once right?

Reply:
Yes, that's what I thought too,
But I wasn't sure if there any functionality I was changing if I just used _triangular_logdet, since initially all the functions had vectorize.
If batching has been removed, I can remove the vmap part entirely and check if that works

Reply:
Sure go for it and see if removing it all together works